crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -33,8 +33,12 @@ def initialize(backup)
     def upgrade
       case @version
       when "1.9"
-        knife_files
-        crowbar_files
+        knife_files19
```

Comment:
feel free to ignore the following taste comment:

can we maybe add another underscore in front of the version like
```ruby
knife_files_19
``` 
? maybe its just me, but it looks somehow ugly ;)